src/formatDate.js

```diff
@@ -8,7 +8,61 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const FROM_SEPARATOR_FORMAT = fromFormat[3];
```

Comment:
Do not use UPPER_CASE for common (dynamic - you've got its value based on function arguments) variables. We can use UPPER_CASE for really constant variables, that might be declared outside of the function and are always the same

Example: const DAYS_IN_WEEK = 7; 

We saved it in the variable so it will be clear what exactly it means in place we use it (instead of some magic 7 number).

But in the current task you don't need to use such variables